tests/python/PyromaBearTest.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+import os.path
+from queue import Queue
+
+from coalib.settings.Section import Section
+from bears.python.PyromaBear import PyromaBear
+
+
+def get_testdir_path(name):
+    return os.path.join(os.path.dirname(__file__), 'pyroma_test_files', name)
+
+
+def get_message_list(results):
+    return [result.message for result in results]
+
+
+class PyromaBearTest(unittest.TestCase):
+
+    def setUp(self):
+        self.section = Section('name')
+        self.queue = Queue()
+        self.file_dict = {}
+
+    def get_results(self, name):
+        try:
+            setup_file = os.path.join(get_testdir_path(name), 'setup.py')
+            with open(setup_file) as file:
+                self.file_dict[setup_file] = file.readlines()
```

Comment:
you don't need to populate file dict with the actual contents since you never use them, right?